Applications/DataExplorer/DataView/GEOModels.h

```diff
@@ -42,6 +42,8 @@ class GEOModels : public QObject, public GeoLib::GEOObjects
 	GEOModels(QObject* parent = 0);
 	~GEOModels();
 
+	void setGEOObjects(GeoLib::GEOObjects* geo_objects);
```

Comment:
Why not initialize the member `_geo_objects` using the constructor and omit the method?